matplotlib2tikz/path.py

```diff
@@ -104,6 +105,9 @@ def draw_pathcollection(data, obj):
         fc = None
     else:
         # gather the draw options
+        is_contour = len(dd) == 1
```

Comment:
Can this go outside of the conditional?

Reply:
Tests pass with this change. I had it in the conditional simply because it always come into the 'else' block in case of contour plot. I simply liked it more that way. Moved the code outside of the conditional block and further down the code.

Reply:
Correction... block must come before extending options.